### PR TITLE
feat(docs): FRMW-198 - Migrate rxStatus docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-198

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxStatus/rxStatus.js [hunk 1]
```diff
@@ -4,7 +4,165 @@
  * @description
  * # rxStatus Component
  *
- * [TBD]
+ * This service is provided as a compliment to the `rxNotify`.  It abstracts out
+ * some of the raw functionality provided by `rxNotify` to make the addition and
+ * removal of single messages easier.
+ *
+ * ## Preparation
```
Comment (ssalinas24) [inline src/rxStatus/rxStatus.js]: @bry4n I confirmed with @CITguy, and I believe the content from here down is supposed to be in the service section. Down around line 198.

Comment (CITguy) [inline src/rxStatus/rxStatus.js]: Correct. Documentation pertains to the service, not the component.

Comment (bry4n) [inline src/rxStatus/rxStatus.js]: Yeah that makes sense. updated.

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**